*: remove `// Author:` comments

tbg opened the pull request:
we have a `CODEOWNERS` file now, and the Author tags were often stale and
likely never useful.

Automated edits using variations of

```
find ./pkg -name '*.h' -exec perl -i -0777 -pe 's~(//\n)?// Author.*\n~~g' '{}' ';'
```

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/17333)
<!-- Reviewable:end -->

petermattis reviewed (approved):
LGTM

I wonder how long they will keep showing up in new files.

tbg committed:
*: remove `// Author:` comments

we have a `CODEOWNERS` file now, and the Author tags were often stale and
likely never useful.

Automated edits using variations of

```
find ./pkg -name '*.h' -exec perl -i -0777 -pe 's~(//\n)?// Author.*\n~~g' '{}' ';'
```

and some manual fixes (for free-standing `// Author:` lines which post removal
upset gofmt).

tschottdorf commented:
Probably a fair bit, but I don't think we'll need to write a linter. I'll just run this again in a few weeks.

couchand commented:
It would be really nice if we could run it against open PRs...

tschottdorf commented:
https://github.com/cockroachdb/cockroach/pull/17346

On Tue, Aug 1, 2017 at 10:28 AM Andrew Couch <notifications@github.com>
wrote:

> It would be really nice if we could run it against open PRs...
>
> —
> You are receiving this because you modified the open/close state.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/17333#issuecomment-319386875>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AE135DyCSnBlPcJY5UB86AJWKwgL5ZN2ks5sTzYagaJpZM4Oo5JH>
> .
>
-- 

-- Tobias
